komfydb/common/int_field.cc

```diff
@@ -43,6 +47,52 @@ Type IntField::GetType() const {
   return Type::INT;
 }
 
+absl::Status IntField::ApplyAggregate(AggregateType aggregate_type,
+                                      Field* other) {
+  // IntField other is assumed here
```

Comment:
If this is assumed here then add `IntField* other = static_cast<IntField*>(other)` (maybe rename the argument to `other_field` or sth).